app/assets/stylesheets/components-vue/_social.scss

```diff
@@ -1,92 +1,118 @@
+
+  @include breakpoint($small) {
+    margin-right: rem-calc(14);
+    width: rem-calc(41); 
+    height: rem-calc(41);
+  }
+}
+
+@mixin social-icon-white ($image) {
+  @include social-basic($image);
+  width: rem-calc(41); height: rem-calc(41);
+
+  &:first-child { 
+    margin-top: rem-calc(10);
+  }
+
+  @include breakpoint($small) {
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+}
+
+//--------------------------------------------------
+// classes
 //--------------------------------------------------
 .social {
 //--------------------------------------------------
-// social
+// following or sharing via social media 
 //--------------------------------------------------
-  &--share {
+  &--media {
+    @include flex;
     position: relative;
 
-    .social {
-      &__trigger {
-        margin: rem-calc(0 20 0 17);
+    .social__icon {
+      &--email { @include social-icon('icons/email.svg'); }
+      &--facebook {
+        @include social-icon-white('icons/facebook-white.svg');
 
-        display: block;
-        
-        @include breakpoint($small) { margin: rem-calc(0 30 0 22); }
-        @include breakpoint($medium) { display: none; }
+        @include breakpoint($small) {
+          @include social-icon('icons/facebook.svg'); 
+        } 
       }
+      &--linkedin { 
+        @include social-icon-white('icons/linkedin-white.svg');
 
-      &__target {
-        @include breakpoint_down($medium) { 
-          background-color: $grey;
-          border-radius: 2px;
-          padding: rem-calc(14 12);
-          display: none;
-          position: absolute;
-          top: rem-calc(40);
-          left: 50%;
-
-          transform: translateX(-50%);
+        @include breakpoint($small) {
+          @include social-icon('icons/linkedin.svg'); 
         }
-        
-        @include breakpoint($small) { top: rem-calc(50); }
+      }
+      &--twitter { 
+        @include social-icon-white('icons/twitter-white.svg');
 
-        @include breakpoint($medium) { 
-          display: block;
-          position: initial;
+        @include breakpoint($small) {
+          @include social-icon('icons/twitter.svg'); 
         }
+      }
+    }
+  }  
 
-        &::before {
-          @include breakpoint_down($medium) { 
-            border-right: rem-calc(12) solid transparent;
-            border-bottom: rem-calc(12) solid $grey;
-            border-left: rem-calc(12) solid transparent;
-            content: '';
-            width: 0; height: 0; 
+  &--media-white {
+    .social__icon {
+      &--email { @include social-icon-white('icons/email-white.svg'); }
+      &--facebook { @include social-icon-white('icons/facebook-white.svg'); }
+      &--linkedin { @include social-icon-white('icons/linkedin-white.svg'); }
+      &--twitter { @include social-icon-white('icons/twitter-white.svg'); }
+    }
+  }
 
-            position: absolute;
-            top: rem-calc(-12);
-            left: 50%;
+  &__popup {
```

Comment:
on mobile view, devtools i think a z-index of 1000 is needed because the bottom edge gets covered by the multilingual component (and possibly others)